src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/entities/fee/FeeDto.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.reform.iacaseapi.domain.entities.fee;
+
+import static java.util.Objects.*;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.math.BigDecimal;
+import lombok.Builder;
+
+@Builder
```

Comment:
Do we need a Builder here?  You have a constructor which has all the args in there and in there you indicate that they are required fields.  So why not just use that since there are only 4 fields?